stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.29.2
+version: 1.29.3
```

Comment:
Please bump the minor version of this chart, as you're adding a new feature.

Reply:
@timstoop Done